### ISSUE TITLE
1.0.0RC4 on OSX El Capitan, OpenSSL version lacking TLS 1.2

### ISSUE THREAD

**Issue by Cpuroast** [sabnzbd/sabnzbd#505]
Just installed 1.0.0RC4 on El capitan using the El capitan version from the DMG and I'm checking the general info page and it says: 

Python Version:	2.7.10 (default, Oct 23 2015, 19:19:21) [GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)]
OpenSSL:	OpenSSL 0.9.8zg 14 July 2015 [t1, v23, v3, v2]
pyOpenSSL:	0.13.1

Wasn't El Capitan supposed to get a newer version of OpenSSL supporting TLS 1.2?
It was mentioned in Issue #401.

**Comment by shypike** [comment]
We hoped for it, but it didn't happen.
Apparently only iPhone security matters these days...
Or we did something stupid.
What happens on your Mac if you start Python and run this:
```
from OpenSSL import SSL
SSL.SSLeay_version(SSL.SSLEAY_VERSION)
```

**Comment by Cpuroast** [comment]
Python 2.7.10 (default, Oct 23 2015, 18:05:06) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.59.5)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from OpenSSL import SSL
>>> SSL.SSLeay_version(SSL.SSLEAY_VERSION)
'OpenSSL 0.9.8zg 14 July 2015'

**Comment by shypike** [comment]
If we create a stand-alone Python package, we'd also lose the connection to the OSX frameworks: meaning no icon in the upper menu bar (at least not without resorting to something like QT).
OTOH it would have the benefit of not having to create a separate version for each and every major OSX release.
It's a mixed bag.

**Comment by thezoggy** [comment]
` pyOpenSSL: 0.13.1` is the issue.. thats older than crap and why i wanted shy to make a new build when i saw it on the last RC. 0.14 is required for tls.. and `0.15.1` is current.

**Comment by thezoggy** [comment]
https://hg.python.org/cpython/file/2.7/Misc/NEWS
looks like there may be a python __2.7.12__ in the near future, which will bump included OpenSSL to 1.0.2g for win/osx.
